### DISCUSSION TITLE
MVP段階でのデプロイ先のチェーンはどこにしましょう？？

### DISCUSSION THREAD

**Discussion by k3nt0w** [CryptoCocoA/matcha-cream#4] [Ideas]
最終的な判断でいいのですがコントラクトのデプロイ先どうしましょうか？

このスレッドで決めて行きましょう。

**Comment by ropital** [comment]
僕はあんまりそれぞれのチェーンの特徴とかあまり詳しくないので、初心者っぽい考えかもですが、ガスコスト低いPolygonとかで良いのかなって思っています！皆さんはどうでしょう？

**Comment by ikisuke** [comment]
Polygonは、Openseaで見れるしガス代が低いのは魅力的なんですが、動いていないみたいなのをよく見かけるのが心配だなと思いました（ ;  ; ）
solidityを使うということでEVM互換のチェーンを主に使うと思いますが、AvalancheやAstarとかも候補としてはいいかもしれませんね！（JPYCもAstar, Avalancheに対応するようです。）
Avalancheの仕組みについては、以下の動画とかが参考になるかもです😀
https://www.youtube.com/watch?v=Crq0i_9xOZU

**Comment by temotibutasan** [comment]
Discordでもありましたが自分が持っているNFTがPolygonが多いのでPolygonだと嬉しいですがAvalancheやAstarも興味はあります。
ikisukeさんから頂いた情報の動画をまず見てみてます（AvalancheAvalanche、Astar関してもよくわかっていないので）
ikisukeさんありがとうございます！
